core/domain/workload.go

```diff
@@ -0,0 +1,5 @@
+package domain
+
+import wssc "github.com/armosec/armoapi-go/apis"
+
+type Workload wssc.WebsocketScanCommand
```

Comment:
question: why `Workload`?

I remember this object being exactly what it is called: a scan command. Personally, I would drop the websocket part, but otherwise it looked to me like a scan command or a scan request.

Reply:
At the beginning I wanted to scan a workload, and since for the moment we wanted to reuse the existing object from the Operator point of view, I have decided to abstract it to my initial thought.
I should probably rename it to `ScanCommand` and at some point use a generic (uncorrelated) representation.